docs/usage/configuration.rst

```diff
@@ -0,0 +1,312 @@
+            }
+        }
+    }
+
+You tell it where to send the check results to (:code:`"nsca.host"`)
+and then define for each check a list of metrics whose values are checked for abnormal values,
+as given by :literal:`{warning,critical}_above`.
+Checks need to be configured as passive checks on the host side,
+otherwise reports will be dropped silently by the host.
+
+Full reference
+--------------
+
+Top-level configuration
+^^^^^^^^^^^^^^^^^^^^^^^
+
+:code:`nsca`
+    A dictionary of :ref:`NSCA host settings`.
+
+:code:`checks`
+    A dictionary of :ref:`check configurations<Check configuration>` by name.
+
+    Example
+        Suppose you have configured a passive check called :literal:`foo` in Centreon/Nagios
+        and want to be alerted when values of either of the metrics :literal:`foo.bar` or :literal:`foo.baz`
+        drop below a threshold.
+        A possible configuration looks like this:
+
+        .. code-block:: json
+
+            {
+                "nsca": {
+                    "host": "nsca.example.org",
+                },
+                "checks": {
+                    "foo": {
+                        "metrics": [
+                            "foo.bar",
+                            "foo.baz"
+                        ],
+                        "warning_below": 10.0,
+                        "critical_below": 5.0
+                    }
+                }
+            }
+
+:code:`reporting_host`
+    Name of the host for which check results are reported, as configured in Nagios/Centreon.
+
+    Default:
+        The current hostname returned by :manpage:`gethostname(2)`.
+
+:code:`resend_interval`
+    .. _global_resend_interval:
+
+    Global default resend interval (see :ref:`Check configuration/resend_interval<resend_interval>`).
+
+    Default
+        :literal:`"3min"`
+
+Check configuration
+^^^^^^^^^^^^^^^^^^^
+
+A single check monitors a set of metrics for abnormal behaviour.
+It continuously consumes new data points for these metrics
+and reports and `overall state`: if values of a single metric exceed their :ref:`allowed range<abnormal-ranges>`
+or there are no new values after :ref:`a certain time<timeout>`
+a state of :literal:`WARNING` or :literal:`CRITICAL` is reported.
+
+:code:`metrics` (list of strings)
+    A list of metrics that should be monitored.
+
+    This list is **mandatory** and required to be *non-empty*.
+
+
+:code:`warning_above`, :code:`warning_below`, :code:`critical_above`, :code:`critical_below` (number)
+    .. _abnormal-ranges:
+
+    Range of value which should trigger a :literal:`WARNING` (resp. :literal:`CRITICAL`) status report to be sent.
+    We call the intervals :math:`[-∞, \mathtt{warning\_below}) \cup (\mathtt{warning\_above}, ∞]` the *warning range*;
+    values within that range trigger a :literal:`WARNING`.
+    The *critical range* is defined similarly.
+
+    The critical range must be contained within the warning range, otherwise the configuration is rejected.
```

Comment:
```suggestion
    The warning range must be contained within the critical range, otherwise, the configuration is rejected.
```

Reply:
Hm... this is technically correct, since the range of values for which warnings are generated contains all values for which `criticals` are generated. But you are right, this sounds confusing.

Reply:
Oh, so I misread "range"...  As this sentence is confusing and redundant to the provided term below. I'd suggest to just leave it out.